x-pack/plugins/infra/public/containers/logs/log_position/log_position_state.ts

```diff
@@ -60,8 +75,23 @@ const useVisibleMidpoint = (middleKey: TimeKeyOrNull, targetPosition: TimeKeyOrN
 };
 
 export const useLogPositionState: () => LogPositionStateParams & LogPositionCallbacks = () => {
+  // Flag to determine if `LogPositionState` has been fully initialized.
+  //
+  // When the page loads, there might be initial state in the URL. We want to
+  // prevent the entries from showing until we have processed that initial
+  // state. That prevents double fetching.
+  const [initialized, setInitialized] = useState<boolean>(false);
```

Comment:
How about `isInitialized` for consistency?

```suggestion
  const [isInitialized, setInitialized] = useState<boolean>(false);
```